lips/lip-2.md

```diff
@@ -0,0 +1,1430 @@
+---
+lip: 2
+title: Libra Roles and Permissions
+author: Tim Zakian, Sam Blackshear
+status: Draft
+type: Informational
+created: 06/26/2020
+---
+
+# Summary
+---
+This LIP describes the conceptual model and implementation of access control on the Libra blockchain.
+
+---
+# Abstract / Motivation
+---
+
+Libra uses a variant of role-based access control (RBAC) to restrict access to sensitive on-chain operations.
+
+A *role* is an entity with some authority in the Libra system. The roles in the initial system are:
+LibraRoot - the root authority of Libra. Controlled jointly by Libra Payment Networks and the Association Council 
+TreasuryCompliance - An entity responsible for day-to-day treasury (e.g. minting, burning), and compliance (e.g., updating on-chain exchange rates, freezing accounts)
+Validator - The on-chain representation of a Libra Association member.
+ValidatorOperator - An entity authorized to operate one or more validator nodes on behalf of an Association member.
+Designated Dealer - An entity that manages fiat transfers to/from the Libra reserve
+ParentVASP - The primary account of a regulated wallet operating on the Libra blockchain
+ChildVASP - A subaccount of a particular parent VASP operating on the Libra blockchain
+
+Every account in the Libra system is created with a single, immutable role that is granted at the time the account is created. Creating an account with a particular role is a privileged oepration (e.g., only an account with the ParentVASP role can create an account the ChildVASP role). In some cases, the role is globally unique (e.g., there is only one account with the LibraRoot role). In other cases, there may be many accounts with the given role (e.g., ChildVASP).
+
+A *permission* is the authority to perform a sensitive action in the Libra system. Each permission may be assigned to one or more roles (usually only one), and each role may have zero or more permissinons. Permissions can be assigned in genesis, upon account creation (most common), or claimed by an existing account with the appropriate role. Like roles, some permissions are globally unique (e.g., the permission to mint a particular currency type) and some are not.
```

Comment:
if there are multiple account for some role (eg ParentVASP), is a permission granted to the role (ParentVASP) or to an account instance of that role (eg coinbase)?

Reply:
The latter--e.g., there is one account with a `ParentVASP` role per real-world VASP.